api/src/main/resources/workbench-api.yaml

```diff
@@ -235,6 +235,35 @@ paths:
       responses:
         204:
           description: Request Received.
+
+  "/v1/getTerraTermsOfServiceStatus":
+    get:
+      tags:
+        - profile
+      description: Retrieves if the logged in user has accepted Terra service or not
+      operationId: getUserTerraTermsOfServiceStatus
+      responses:
+        200:
+          description: Returns false if user is required to accept Terra Terms of Service, else true
```

Comment:
this is a little awkward.  How about: returns true if the user is current

Reply:
Updated